fpdf/html.py

```diff
@@ -595,16 +595,20 @@ def handle_starttag(self, tag, attrs):
             self.pdf.insert_toc_placeholder(
                 self.render_toc, pages=int(attrs.get("pages", 1))
             )
+        if tag == "sup":
+            self.pdf.char_vpos = "SUP"
+        if tag == "sub":
+            self.pdf.char_vpos = "SUB"
 
     def handle_endtag(self, tag):
         # Closing tag
         LOGGER.debug("ENDTAG %s", tag)
         if tag in self.heading_sizes:
             self.heading_level = None
             face, size, color = self.font_stack.pop()
+            self.pdf.ln(self.h * 1.2)  # 20 % more space below heading
```

Comment:
Could you move this constant in a class property please?

Reply:
ok